Azure/azure-cli
# [Backup] `az backup recoveryconfig show`: Add new parameter `--target-instance-name` to specify the target instance name for the restore operation

**Related command**


**Description**


Users can now use the optional `--target-instance-name` parameter while running the `az backup recoveryconfig show`. This unblocks customers in certain edge-case scenarios where the target instance name is the only distinguishing factor between two backup instances.

**Testing Guide**

It is now possible to add the `--target-instance-name` parameter to commands. There should not be any visible change in behaviour for most use cases.

Please ref: IcM 374840256 for more details.

**History Notes**


[Backup] `az backup recoveryconfig show`: Add new parameter `--target-instance-name` to specify the target instance name for the restore operation

---

This checklist is used to make sure that common guidelines for a pull request are followed.
